app/javascript/components/reconfigure-vm-form/reconfigure-form-fields.js

```diff
@@ -0,0 +1,152 @@
+import { componentTypes } from '@@ddf';
+import { dataTableHeaders, restructureOptions, socketChange } from './helper';
+import { setDiskData } from './helpers/disk';
+import { setNetworkData } from './helpers/network';
+import { setDrivesData } from './helpers/drive';
+import { TYPES } from './helpers/general';
+
+const memoryField = (roles) => ({
+  component: 'switch',
+  name: 'cb_memory',
+  label: __('Memory'),
+  onText: __('Yes'),
+  offText: __('No'),
+  autoFocus: true,
```

Comment:
we can remove the `autofocus:true` since the setState is rendering the form and focus goes to the top of the page